smoke-test/src/test/java/org/opennms/smoketest/SystemReportPageIT.java

```diff
@@ -0,0 +1,55 @@
+package org.opennms.smoketest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Objects;
+
+
+public class SystemReportPageIT extends UiPageTest  {
+
+
+    private Page uiPage;
+
+    @Before
+    public void setUp() throws Exception {
+        uiPage = new SystemReportPageIT.Page(getBaseUrlInternal());
+        uiPage.open();
+    }
+
+    @Test
+    public void systemReport() throws IOException, InterruptedException {
+
+
+        WebElement filenameInput = driver.findElement(By.id("filename"));
+        filenameInput.sendKeys("abc@!321.txt");
+        WebElement generateButton = driver.findElement(By.xpath("//input[@class='btn btn-secondary' and @value='Generate System Report']"));
+        generateButton.click();
+        execute(() -> {
+            pageContainsText("abc321.txt");
+            return null;
+        });
+
+    }
+
+    private class Page {
+        private final String url;
+
+        public Page(String baseUrl) {
+            this.url = Objects.requireNonNull(baseUrl) + "opennms/admin/support/systemReport.htm";
+        }
+        public Page open() {
+            driver.get(url);
+            System.out.println(url);
+            new WebDriverWait(driver, Duration.ofSeconds(10)).until(pageContainsText("Choose which plugins to enable:"));
```

Comment:
I don't find this `Choose which plugins to enable:` on `opennms/admin/support/systemReport.htm`

Reply:
![Screenshot from 2024-12-19 14-44-59](https://github.com/user-attachments/assets/000b1e26-8963-4e76-9772-0cb8115762ec)

It is before checkboxes ,In first line.